semantic_segmentation/code/pretrain.py

```diff
@@ -1,21 +1,14 @@
-from comet_ml import Experiment
 import torch
-import torch.nn as nn
 import torchvision
-from torchvision import transforms
 from torch.utils.data import DataLoader
-import torch.nn.functional as F
 import models
-import argparse
+from args import get_args
```

Comment:
will that do `import argparse` also automatically? Please check once.

Reply:
yes!